### PR TITLE
Add Test for React Edit Comment

### PR DESCRIPTION
**Pull request by noi5e**
Part of the React rewrite of the comment system. See #9365

Added system test for editing comments that handles both React and Rails comment systems.

Also tweaked the `CommentToolbarButton` component so that it displays a class for `edit-comment-btn`s

---
(This issue is part of the larger Comment Editor Overhaul Project with Outreachy. Refer to Planning Issue #9069 for more context)

### CODE REVIEW

File: test/system/comment_test.rb [hunk 1]
```diff
@@ -247,28 +276,6 @@ def get_path(page_type, path)
       assert_selector('.btn[data-original-title="Help"]', count: 1)
     end
 
-    test "#{page_type_string}: edit comment" do
-      nodes(node_name).add_comment({
```
Comment (noi5e) [inline test/system/comment_test.rb:251]: Deleting this system test, as it's replaced by the new one.

**Comment by jywarren** [comment]
This looks awesome. Thanks @noi5e!

I did want to note i saw a warning in the Jenkins build log:

```
WARNING in ./app/javascript/components/REACT_COMMENTS.md 1:0
Module parse failed: Unexpected character '#' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> # `React Commenting System`
| 
| **Last Updated: 3/13/2021**
 @ ./app/javascript/components sync ^\.\/.*$ ./REACT_COMMENTS.md
 @ ./app/javascript/packs/server_rendering.js
```

I want to say i don't think anything broke, but just noting in case it's an issue to look out for.

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Looks awesome. Merged! Pls give it a spin at https://stable.publiclab.org once it builds -- thanks a ton, @noi5e !!!

**Comment by noi5e** [comment]
@jywarren: A little more about that error that you mentioned also [here](https://github.com/publiclab/plots2/pull/9640#issuecomment-852286327)

```
WARNING in ./app/javascript/components/REACT_COMMENTS.md 1:0
Module parse failed: Unexpected character '#' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> # `React Commenting System`
| 
| **Last Updated: 3/13/2021**
 @ ./app/javascript/components sync ^\.\/.*$ ./REACT_COMMENTS.md
 @ ./app/javascript/packs/server_rendering.js
```

This is occurring because webpacker is trying to parse `./app/javascript/components/REACT_COMMENTS.md` (which is written in Markdown), as if it's a `.jsx` file.

webpacker encounters `#`, which is the first character in `REACT_COMMENTS.md`. It throws an error, because this isn't syntactical in `.jsx`.

It's a minor thing and low-priority IMO, but would be good to take care of down the line. The webpacker config needs to be altered. I'll put it on my to-do.

**Comment by jywarren** [comment]
Thank you this makes total sense. 💫💫💫

**Pull request merged**